robotiq_driver/tests/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Add support for GTest.
+find_package(ament_cmake_gtest REQUIRED)
+find_package(ament_cmake_gmock REQUIRED)
+find_package(ament_lint_auto REQUIRED)
+find_package(ros2_control_test_assets REQUIRED)
```

Comment:
This needs to be added to the package.xml as a `<test_depend>`

Reply:
Done. Thanks.